homeassistant/components/overkiz/__init__.py

```diff
@@ -45,15 +53,42 @@ class HomeAssistantOverkizData:
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Overkiz from a config entry."""
-    username = entry.data[CONF_USERNAME]
-    password = entry.data[CONF_PASSWORD]
-    server = SUPPORTED_SERVERS[entry.data[CONF_HUB]]
-
-    # To allow users with multiple accounts/hubs, we create a new session so they have separate cookies
-    session = async_create_clientsession(hass)
-    client = OverkizClient(
-        username=username, password=password, session=session, server=server
-    )
+
+    client = None
+
+    # Local API vs Cloud API
+    # TODO add a helper function to see if local API
```

Comment:
What about adding this directly within pyoverkiz? Within the client we can try to detect if we the local mode is enabled, and if yes, use it.

Reply:
@tetienne I would like that, just not sure yet how we should do this in the best way. Currently we do the following check: 
https://github.com/iMicknl/python-overkiz-api/blob/4a5ec7dc896fb16f85e9979aac46dc1ebbe0a081/pyoverkiz/client.py#L157